Containers/talk/Dockerfile

```diff
@@ -1,44 +1,45 @@
 FROM nats:2.9.19-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
+
+FROM alpine:3.18.2 as janus
+ARG JANUS_VERSION=v0.13.4
+WORKDIR /src
+RUN apk add --no-cache \
+        ca-certificates \
+        git autoconf automake build-base pkgconfig libtool util-linux \
+        glib-dev zlib-dev openssl-dev jansson-dev libnice-dev libconfig-dev libsrtp-dev gengetopt-dev \
+        libwebsockets-dev; \
```

Comment:
we maybe need to change to optional modules

Reply:
the structureag image also contains: `curl-dev libogg-dev`

Reply:
ogg adds `Voice Mail` and curl `Sample event handler`/`Sample event handler`

Reply:
If I read https://github.com/strukturag/nextcloud-spreed-signaling#setup-of-janus, we don't need those three modules

Reply:
all right, then lets remove them.

Reply:
ah I see, sorry